worlds/stardew_valley/docs/en_Stardew Valley.md

```diff
@@ -124,6 +124,6 @@ List of supported mods:
 
 ## Multiplayer
 
-You cannot play an Archipelago Slot in multiplayer at the moment. There is no short-terms plans to support that feature.
+You cannot play an Archipelago Slot in multiplayer at the moment. There is no short-term plans to support that feature.
```

Comment:
```suggestion
You cannot play an Archipelago Slot in multiplayer at the moment. There are no short-term plans to support that feature.
```
Should also have a plural verb while we're at it.

Reply:
yeah probably :)